### DISCUSSION TITLE
API support for `SafeLinksPolicy`

### DISCUSSION THREAD

**Discussion by Subhajit97** [microsoftgraph/msgraph-sdk-go#262] [Q&A] [answered]
Hi!
We are trying to verify whether the Safe Links policy is enabled, or not.
I can see an equivalent Powershell command to do that:
`Get-SafeLinksPolicy -Identity "Policy Name"`

Is there any equivalent API in this SDK?

**Comment by baywet** [comment, marked as answer]
Hi @Subhajit97 
This repository (and issues/discussions) is dedicated to the SDK itself. For questions about the APIs, the best is to post [on the dedicated Microsoft Q&A forum](https://docs.microsoft.com/en-us/answers/topics/microsoft-graph-security.html) to get an answer from the product team.
